.circleci/config.yml

```diff
@@ -345,3 +345,31 @@ workflows:
                 - << pipeline.parameters.dev_git_branch >>
                 - << pipeline.parameters.staging_git_branch >>
                 - << pipeline.parameters.prod_git_branch >>
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 5 * * *"
```

Comment:
I think it would fit better with our team structure to discover vulnerabilities at mid-day and on working days rather than at 1am every day. How would you feel about  "0 17 * * 1-5"? I'm not opposed to asking someone to work on the weekend if a security vulnerability is found, but we don't have an alerting system in place to tell engineers when this scheduled run has failed, so I'm not seeing the value here.  Also, since we're re-running the build aren't we potentially updating packages to a new version, a new version that could bring with it a new security failure?

Reply:
`yarn deps` will only install the versions in `yarn.lock` so no new versions should ever be installed by CircleCI.

I'm good with only running on weekdays, but how about in that case we schedule it for 9am eastern and then make failures a "first thing someone picks up today" kind of thing. Does that work for you?

Reply:
> yarn deps will only install the versions in yarn.lock so no new versions should ever be installed by CircleCI.

Oh of course, I forgot about that.

9am EST is fine with me. But FYI, I don't start work until 9am CST so if anyone slacks me before that I may not notice. Josh is CST too though I don't know what time he starts his day.

Reply:
We're not going to be running 15 minute firedrills on failures. I'm more thinking that we should have them run before anyone starts because it's less impactful to fix something when you start your day (whenever that is) than to get interrupted in the middle.